### ISSUE TITLE
Arena for orcs

### ISSUE THREAD

**Issue by karnedi** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth#1082]
In the culture of orcs, as is known, there are fights in arenas. It would be nice to add the great arena as a great structure to the building opportunities for orcs
----------------------------------------------------------------------------------------------------------

**Comment by babosh-gh** [comment]
I actually like that idea.

Also, if you want to follow WoW gameplay/lore a bit, you could also make a cross-faction warrior-society for arena games imo.

**Comment by zumbak04** [comment]
> Also, if you want to follow WoW gameplay/lore a bit, you could also make a cross-faction warrior-society for arena games imo.

@babosh-gh you mean Brawler's Guild?

**Comment by babosh-gh** [comment]
@zumbak04 No, I meant the pvp arena system ( i know those arent 1v1 matches )

**Comment by zumbak04** [comment]
> @zumbak04 No, I meant the pvp arena system ( i know those arent 1v1 matches )

Doubt there is lore kind of society behind arena.

**Comment by babosh-gh** [comment]
@zumbak04 There is quite a lot of lore behind pvp arenas. A lot more lore than behind brawler's guild, actually.

Varian Wrynn was a gladiator fighter for quite a while in an orcish arena pit, lead by Rehgar Earthfury.
Varian earned the respect of the orcish fighters of the arena because of his skill in combat, and also made friends there with Valeera and Broll Bearmantle, and those guys remained his friends to his last day.

So imo this part of the lore justifies the creation of a warrior society which can be used by different races (albeit i think it would make the most sense if its created in-game only after the formation of the horde and the alliance)

**Comment by zumbak04** [comment]
> @zumbak04 There is quite a lot of lore behind pvp arenas. A lot more lore than behind brawler's guild, actually.
> 
> Varian Wrynn was a gladiator fighter for quite a while in an orcish arena pit, lead by Rehgar Earthfury.
> Varian earned the respect of the orcish fighters of the arena because of his skill in combat, and also made friends there with Valeera and Broll Bearmantle, and those guys remained his friends to his last day.
> 
> So imo this part of the lore justifies the creation of a warrior society which can be used by different races (albeit i think it would make the most sense if its created in-game only after the formation of the horde and the alliance)

This society is called [the Crimson Ring](https://wow.gamepedia.com/Crimson_Ring). They owned the Orgrimmar Arena as some point before the Brawler's Guild.
